data/blog/renewing-our-commitment-to-bitcoin.mdx

```diff
@@ -0,0 +1,61 @@
+authors: ['arvin', 'dergigi', 'btcschellingpt']
+images: ['/static/images/blog/64-renewing-our-commitment-to-bitcoin.jpg']
+summary: "Renewing our support for projects advancing Bitcoin's future."
+---
+
+Over the past two years, we have made [41 announcements](/tags/bitcoin) from our
+General Fund highlighting our commitment to supporting individual contributors
+and projects working to grow the Bitcoin ecosystem. This ecosystem is built on
+the innovation, dedication, collaboration, and the tireless efforts of
+open-source contributors.
+
+Bitcoin's influence is ever expanding—not just for individuals reclaiming
+financial independence, but also affecting public markets, creating new
+financial instruments, and [other
+innovations](https://www.accesswire.com/946068/newmarket-launches-battery-finance-a-pioneering-asset-management-and-specialty-finance-platform-built-on-bitcoin-completes-inaugural-credit-investment)
+that bridge traditional finance with this pristine digital asset class. All this
+demonstrates that the Bitcoin experiment is working as intended: it is slowly
+but surely reshaping how money operates.
+
+Beyond our previous announcements, our commitment runs deeper. We've doubled
+down our support for 9 exceptional projects advancing the Bitcoin ecosystem.
+These projects remind us of [our purpose](/mission): to ensure that open-source
+builders and innovators in the Bitcoin space have the resources they need to
+push boundaries. It's an honor to provide additional grants to the following
+projects:
+
+- [BTCPay Server](/blog/bitcoin-grants-july-2023#btcpay-server)
+- [Stratum
+  V2](/blog/bitcoin-and-nostr-grants-august-2023#stratum-v2-testing--benchmarking-tool)
+- [Raspiblitz](/blog/bitcoin-grants-july-2023#raspiblitz)
+- [LNbits](/blog/bitcoin-and-nostr-grants-august-2023#lnbits)
+- [Vexl](/blog/bitcoin-grants-december-2023#vexl)
+- [Blixt](/blog/bitcoin-grants-july-2023#blixt-wallet)
+- [Krux](/blog/bitcoin-grants-december-2023#krux)
+- [Bitaxe](/blog/bitcoin-grants-feb-2024#the-bitaxe)
+- [Labelbase](/blog/bitcoin-grants-july-2023#labelbase)
+
+In addition to these projects, our commitment and long-term support to [key
+infrastructure engineers](/tags/core) remains unchanged.
+
+As Bitcoin continues to redefine the future of money, the work of these
+individuals and projects are the foundation for its progress. Their dedication
+to free and open-source development ensures that Bitcoin remains secure,
+decentralized, and accessible to everyone.
+
+We're grateful to our donors, whose generosity makes this support possible. Your
+contributions directly empower the builders shaping Bitcoin's path forward. If
+you share our vision, then please consider making a donation to the [General
+Fund](/funds/general):
+
+<center>
+    <DonateToGeneralFundButton/>
+</center>
+
+If you are an open-source developer working on Bitcoin, do not hesitate to apply
+for a grant: [https://opensats.org/apply/grant](https://opensats.org/apply/grant)
```

Comment:
Consider turning this apply link into a button similar to the Donate to the General Fund button.

Reply:
Adjusted to have `apply for a grant` be the hyperlink.